<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>v11でPythonデータソースが追加できない</title>
    <link>https://discuss.redash.io/t/v11-python/9576</link>
    <description>redash 11.0.0-dev をECS Fargate で実行しています。
ECS上のコンテナの環境変数には
 REDASH_ADDITIONAL_QUERY_RUNNER=redash.query_runner.python
を設定しているのですが、データソースでPythonが出てきません。
v11で何か設定が変わったのでしょうか？
どなたか教えていただけると助かります。
どうぞよろしくお願いいたします。</description>
    
    <lastBuildDate>Mon, 17 Jan 2022 07:35:09 +0000</lastBuildDate>
    <category>Japanese</category>
    <atom:link href="https://discuss.redash.io/t/v11-python/9576.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>v11でPythonデータソースが追加できない</title>
        <dc:creator><![CDATA[kaori]]></dc:creator>
        <description><![CDATA[
            <aside class="quote no-group" data-username="kaori" data-post="1" data-topic="9576">
<div class="title">
<div class="quote-controls"></div>
<img alt="" width="20" height="20" src="https://sjc6.discourse-cdn.com/standard17/user_avatar/discuss.redash.io/kaori/40/3547_2.png" class="avatar"> kaori:</div>
<blockquote>
<p>REDASH_ADDITIONAL_QUERY_RUNNER</p>
</blockquote>
</aside>
<p>×：REDASH_ADDITIONAL_QUERY_RUNNER<br>
〇：REDASH_ADDITIONAL_QUERY_RUNNERS</p>
<p>typo でした。すいません、ありがとうございました。</p>
          <p><a href="https://discuss.redash.io/t/v11-python/9576/4">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/v11-python/9576/4</link>
        <pubDate>Mon, 17 Jan 2022 07:35:09 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-9576-4</guid>
        <source url="https://discuss.redash.io/t/v11-python/9576.rss">v11でPythonデータソースが追加できない</source>
      </item>
      <item>
        <title>v11でPythonデータソースが追加できない</title>
        <dc:creator><![CDATA[kaori]]></dc:creator>
        <description><![CDATA[
            <p>返信ありがとうございます。<br>
dockerというか、ECSは何度かタスクが起動しなおしています。<br>
serverとworker両方に同じ環境変数が入っています。<br>
旧バージョンからのアップグレードではなく、今回が新規作成となります。<br>
どうぞよろしくお願いいたします。</p>
          <p><a href="https://discuss.redash.io/t/v11-python/9576/3">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/v11-python/9576/3</link>
        <pubDate>Thu, 02 Dec 2021 04:31:48 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-9576-3</guid>
        <source url="https://discuss.redash.io/t/v11-python/9576.rss">v11でPythonデータソースが追加できない</source>
      </item>
      <item>
        <title>v11でPythonデータソースが追加できない</title>
        <dc:creator><![CDATA[yishikawa]]></dc:creator>
        <description><![CDATA[
            <p><a class="mention" href="https://discuss.redash.io/u/kaori">@kaori</a><br>
dockerは再起動しましたか？<br>
serverとworker両方に同じ環境変数を追加していますか？<br>
旧バージョンからのアップグレードをしていますか？<br>
そのデータソースを利用したQueryがあれば、そこではどのようなエラーが出ますか？</p>
          <p><a href="https://discuss.redash.io/t/v11-python/9576/2">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/v11-python/9576/2</link>
        <pubDate>Thu, 02 Dec 2021 02:36:22 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-9576-2</guid>
        <source url="https://discuss.redash.io/t/v11-python/9576.rss">v11でPythonデータソースが追加できない</source>
      </item>
      <item>
        <title>v11でPythonデータソースが追加できない</title>
        <dc:creator><![CDATA[kaori]]></dc:creator>
        <description><![CDATA[
            <p>redash 11.0.0-dev をECS Fargate で実行しています。<br>
ECS上のコンテナの環境変数には<br>
REDASH_ADDITIONAL_QUERY_RUNNER=redash.query_runner.python<br>
を設定しているのですが、データソースでPythonが出てきません。<br>
v11で何か設定が変わったのでしょうか？<br>
どなたか教えていただけると助かります。<br>
どうぞよろしくお願いいたします。</p>
          <p><a href="https://discuss.redash.io/t/v11-python/9576/1">Read full topic</a></p>
        ]]></description>
        <link>https://discuss.redash.io/t/v11-python/9576/1</link>
        <pubDate>Mon, 29 Nov 2021 23:35:10 +0000</pubDate>
        <guid isPermaLink="false">discuss.redash.io-post-9576-1</guid>
        <source url="https://discuss.redash.io/t/v11-python/9576.rss">v11でPythonデータソースが追加できない</source>
      </item>
  </channel>
</rss>
